source/voice_control/android.markdown

```diff
@@ -2,18 +2,45 @@
 title: "Assist on Android"
 ---
 
+## Assist on Android phones
+
+Assist is available on Android phones and tablets using the [Home Assistant Companion App](https://companion.home-assistant.io/docs/getting_started/) for Android.
+
+### Prerequisites
+
+- The devices you want to control with Assist are [exposed to Assist](/voice_control/voice_remote_expose_devices/)
+
+### Setting up Home Assistant Assist as default assistant app on an Android phone
+
+This section was written using an Android 13. Depending on your flavor of Android, the exact steps may vary.
+
+To define Home Assistant Assist as default assistant app, follow these steps:
+
+1. On Android, go to **Settings** > **Apps** > **Default apps**.
+2. Under **Digital assistant app**, select **Default digital assistant app**.
+   - On some Android versions, this might be under **Digital & voice input** > **Default digital assistant app** > **Voice Assistant**.
+3. Select **Home Assistant**.
+   - On some Android versions, you might be able to change the assistants language settings here by selecting the cogwheel.
+4. Go back one step. The **Default digital assistant app** should now show *Home Assistant* as the default.
+5. Leave the **Settings**.
+6. Start Assist using the gesture to start an assistant. The gesture may differ depending on the version.
+   - Swipe from the bottom left corner.
+   - Long press the power button.
+
+## Assist on Wear OS
+
 Assist is available on Wear OS using the [Home Assistant Companion App](https://companion.home-assistant.io/) for Android and "Assist" tile.
 
 <lite-youtube videoid="Dr_ZCbt8w5k" videotitle="Assist on Wear OS"></lite-youtube>
 
-## Installation
+### Installation
 
 After [installing the companion app](https://companion.home-assistant.io/docs/getting_started/) and connecting it to your Home Assistant, visit your watch companion app or the Wear OS app and click "Add tile" inside the Tiles area. Select the "Assist" tile to add it to your watch:
```

Comment:
It's also possible to add a "complication" within the watch face editing screen.

Plus, you can set Assist as the default digital assistant and hold the secondary button to bring it up (even if the watch is asleep).

Reply:
Could we do the Wear OS procedure in a separate PR? I haven't started on that yet.

Reply:
Thank you for the input! Added separate procedures on this.